src/triangulation.rs

```diff
@@ -322,6 +322,15 @@ pub trait Triangulation: Default {
         self.s().fixed_vertices()
     }
 
+    /// Get a vertex by its index
+    fn get_vertex(
```

Comment:
Please also change the type to `FixedVertexHandle`. It prevents callers from accidentally passing in other handles (e.g. edge handles) and is more consistent with the rest of the API